3p/messaging.js

```diff
@@ -14,6 +14,52 @@
  * limitations under the License.
  */
 
+import './polyfills';
+import {dev} from '../src/log';
+
+/** @const */
+const AMP_MESSAGE_PREFIX = 'amp-';
+
+/** @enum {string} */
+export const MessageType = {
```

Comment:
Eventually AMP runtime will need to handle all those message types because inabox can be inside an AMP page.

So it's reasonable to put all types in one place to avoid conflicts. 
On the other side, we're just sharing a enum, which should not bloat inabox-host code.